examples/kitchen-sink/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# @turnkey/kitchen-sink
+
+## 0.1.2
+
+### Patch Changes
+
+- Updated dependencies [8bea78f]
+  - @turnkey/sdk-browser@1.10.0
+  - @turnkey/sdk-react@1.2.0
```

Comment:
do we need changelogs for all examples? I don't think this adds much but I'm okay either way

Reply:
I've removed the changelogs for examples in the last release

Reply:
Removed!